cinema/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+from .models import Movie
+
+class MovieSerializer(serializers.Serializer):

```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for a more concise and efficient implementation. `ModelSerializer` automatically generates the `create` and `update` methods and maps model fields to serializer fields.